pkg/adapter/anchore/adapter.go

```diff
@@ -354,53 +364,55 @@ func BuildHarborVulnerabilityReport(
 	imageRepository string,
 	imageDigest string,
 	includeDescriptions bool,
-	clientConfig *client.ClientConfig,
+	clientConfig *client.Config,
```

Comment:
We're confident these aren't impacting something?

Reply:
Yeah functionally we're all good and nobody is importing this package for use in other code but if we decide that's a concern when we do the next release we'll mark as a major